<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasteServiceEditDialog</class>
 <widget class="QDialog" name="PasteServiceEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paste service</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../HaveClip.qrc">
    <normaloff>:/gfx/HaveClip_128.png</normaloff>:/gfx/HaveClip_128.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Service</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="pasteServiceComboBox">
       <item>
        <property name="text">
         <string>Stikked</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Pastebin.com</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HaveSnippet</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelLabel">
       <property name="text">
        <string>Label</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="labelLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="pasteStackedWidget">
     <property name="currentIndex">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../HaveClip.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PasteServiceEditDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PasteServiceEditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pasteServiceComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>pasteStackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>213</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>188</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
